Settings.c

```diff
@@ -541,6 +541,9 @@ Settings* Settings_new(unsigned int initialCpuCount, Hashtable* dynamicColumns)
       this->changed = true;
       Settings_read(this, SYSCONFDIR "/htoprc", initialCpuCount);
    }
+   if (!ok) {
+      Settings_defaultMeters(this, initialCpuCount);
+   }
```

Comment:
What's the reason here?

Reply:
Try running `HTOPRC=./htoprc_new ./htop`.
The first time the header is empty, cause no default meters are added.
The second time meters are added, cause a configuration file already exists so a different path is taken.